src/views/products/add.vue

```diff
@@ -149,6 +148,9 @@ export default {
 						this.$store.commit('products/ADD', response.data.data);
 						this.name = '';
 						this.price = 1;
+						this.brand="";
```

Comment:
Don't make a `brand`, `measurementUnit` and `category` blank after save an product, this dispatch the form validation 
![image](https://user-images.githubusercontent.com/35310226/197370665-f561200e-6e41-4d42-821c-e1ae9968a60f.png)

Reply:
This is done